mgmtsystem_action/reports/mgmtsystem_action_report.xml

```diff
@@ -33,10 +33,7 @@
     <record id="action_report_mgmtsystem_action" model="ir.actions.act_window">
         <field name="name">Management System Analysis</field>
         <field name="res_model">mgmtsystem.action.report</field>
-        <field name="view_type">form</field>
         <field name="view_mode">pivot,graph</field>
-        <field name="context">{"search_default_year":1,"search_default_User":1,"search_default_This Month":1,'group_by_no_leaf':1,'group_by':[]}</field>
```

Comment:
Why do you remove default filters?

Reply:
Because it doesn't seem to work

Reply:
Well, they work if they are defined as filters in the search view. But if they have disappeared, OK to remove them.